common/src/models/deployment.rs

```diff
@@ -84,9 +88,33 @@ pub fn get_deployments_table(deployments: &Vec<Response>, service_name: &str, pa
                 Cell::new("Last updated")
                     .set_alignment(CellAlignment::Center)
                     .add_attribute(Attribute::Bold),
+                Cell::new("Commit ID")
+                    .set_alignment(CellAlignment::Center)
+                    .add_attribute(Attribute::Bold),
+                Cell::new("Commit Message")
+                    .set_alignment(CellAlignment::Center)
+                    .add_attribute(Attribute::Bold),
+                Cell::new("Branch")
+                    .set_alignment(CellAlignment::Center)
+                    .add_attribute(Attribute::Bold),
+                Cell::new("Dirty")
+                    .set_alignment(CellAlignment::Center)
+                    .add_attribute(Attribute::Bold),
             ]);
 
         for deploy in deployments.iter() {
+            let truncated_commit_id = deploy
+                .git_commit_id
+                .as_ref()
+                .map_or(String::from("No Data"), |val| val.chars().take(7).collect());
```

Comment:
nit: Maybe we can use "N/A" instead? 

Also, it would be good to have it as a constant, used in the rest of the places it's needed.